code/modules/power/supermatter/supermatter.dm

```diff
@@ -89,6 +89,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	desc = "A strangely translucent and iridescent crystal."
 	icon = 'icons/obj/supermatter.dmi'
 	icon_state = "darkmatter"
+	layer = 4
```

Comment:
Why change the layer?
Also, use the macros

Reply:
layer change is so that people dont walk over it since its 2 tiles tall, so they walk under it